platforms/documentation/docs/src/docs/userguide/authoring-builds/gradle-properties/service_injection.adoc

```diff
@@ -272,6 +279,15 @@ include::sample[dir="snippets/providers/services/kotlin",files="build.gradle.kts
 include::sample[dir="snippets/providers/services/groovy",files="build.gradle[tags=tooling-model]"]
 ====
 
+You can learn more at <<tooling_api.adoc#tooling_api,Tooling API>>.
+
+[[testeventreporterfactory]]
+== 10. `TestEventReporterFactory`
```

Comment:
Might be worth mentioning this is incubating

Reply:
Added incubating warning.